src/modules/launcher/PowerLauncher/App.xaml.cs

```diff
@@ -173,13 +173,17 @@ protected virtual void Dispose(bool disposing)
                     Log.Info("|App.OnExit| Start PowerToys Run Exit----------------------------------------------------  ");
                     if (disposing)
                     {
-                        _themeManager.ThemeChanged -= OnThemeChanged;
-                        API.SaveAppAllSettings();
+                        if (_themeManager != null)
+                        {
+                            _themeManager.ThemeChanged -= OnThemeChanged;
+                        }
+
+                        API?.SaveAppAllSettings();
                         PluginManager.Dispose();
```

Comment:
@somil55 What about PluginManager?

Reply:
@ryanbodrug-microsoft It's a static class, so no PluginManager instance that is initialized.